### ISSUE TITLE
REGRESSION: Offline validation fails

### ISSUE THREAD

**Issue by stevespringett** [CycloneDX/cyclonedx-core-java#87]
Validating CycloneDX SBOMs in an offline environment fails.

```
com.networknt.schema.JsonSchemaException: java.net.UnknownHostException: cyclonedx.org
        at com.networknt.schema.JsonSchemaFactory.getSchema(JsonSchemaFactory.java:361)
        at com.networknt.schema.RefValidator.getRefSchema(RefValidator.java:83)
        at com.networknt.schema.RefValidator.<init>(RefValidator.java:46)
        at jdk.internal.reflect.GeneratedConstructorAccessor8.newInstance(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at com.networknt.schema.ValidatorTypeCode.newValidator(ValidatorTypeCode.java:131)
        at com.networknt.schema.JsonMetaSchema.newValidator(JsonMetaSchema.java:351)
        at com.networknt.schema.ValidationContext.newValidator(ValidationContext.java:54)
        at com.networknt.schema.JsonSchema.read(JsonSchema.java:206)
        at com.networknt.schema.JsonSchema.getValidators(JsonSchema.java:356)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:227)
        at com.networknt.schema.PropertiesValidator.validate(PropertiesValidator.java:70)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.JsonSchemaRef.validate(JsonSchemaRef.java:49)
        at com.networknt.schema.RefValidator.validate(RefValidator.java:138)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.PropertiesValidator.validate(PropertiesValidator.java:70)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.ItemsValidator.doValidate(ItemsValidator.java:93)
        at com.networknt.schema.ItemsValidator.validate(ItemsValidator.java:80)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.PropertiesValidator.validate(PropertiesValidator.java:70)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.JsonSchemaRef.validate(JsonSchemaRef.java:49)
        at com.networknt.schema.RefValidator.validate(RefValidator.java:138)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.ItemsValidator.doValidate(ItemsValidator.java:93)
        at com.networknt.schema.ItemsValidator.validate(ItemsValidator.java:80)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.PropertiesValidator.validate(PropertiesValidator.java:70)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.PropertiesValidator.validate(PropertiesValidator.java:70)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.JsonSchemaRef.validate(JsonSchemaRef.java:49)
        at com.networknt.schema.RefValidator.validate(RefValidator.java:138)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.ItemsValidator.doValidate(ItemsValidator.java:93)
        at com.networknt.schema.ItemsValidator.validate(ItemsValidator.java:80)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.PropertiesValidator.validate(PropertiesValidator.java:70)
        at com.networknt.schema.JsonSchema.validate(JsonSchema.java:229)
        at com.networknt.schema.BaseJsonValidator.validate(BaseJsonValidator.java:96)
        at org.cyclonedx.parsers.JsonParser.validate(JsonParser.java:234)
        at org.cyclonedx.parsers.JsonParser.validate(JsonParser.java:219)
        at org.cyclonedx.parsers.JsonParser.validate(JsonParser.java:124)
        at org.cyclonedx.parsers.JsonParser.isValid(JsonParser.java:267)
        at org.cyclonedx.BomJsonGeneratorTest.schema12GenerationTest(BomJsonGeneratorTest.java:46)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
        at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
        at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
        at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
        at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
        at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
        at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.net.UnknownHostException: cyclonedx.org
        at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220)
        at java.base/java.net.Socket.connect(Socket.java:609)
        at java.base/java.net.Socket.connect(Socket.java:558)
        at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:182)
        at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:474)
        at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:569)
        at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:242)
        at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:341)
        at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:362)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1253)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1015)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1592)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)
        at com.networknt.schema.uri.URLFetcher.openConnectionCheckRedirects(URLFetcher.java:57)
        at com.networknt.schema.uri.URLFetcher.fetch(URLFetcher.java:43)
        at com.networknt.schema.uri.URISchemeFetcher.fetch(URISchemeFetcher.java:50)
        at com.networknt.schema.JsonSchemaFactory.getSchema(JsonSchemaFactory.java:337)
        ... 81 more
```

**Comment by stevespringett** [comment]
@DarthHater Any ideas?

**Comment by stevespringett** [comment]
Resolved in 4.1.1
